backend/app/policies/enrollment_policy.rb

```diff
@@ -43,6 +43,16 @@ def copy?
       return false
     end
 
+    if record.copied_from_enrollment_id?
+      @error_message_key = :copy_enrollment_is_a_copy
```

Comment:
```suggestion
      @error_message_key = :enrollment_is_a_copy
```

ce `copy_` devant ne sert à rien, la clé me semble plus explicite comme ça

Reply:
C'était pour suivre la convention de naming des clefs précédentes, dans ce cas-là même `copy_user_is_not_demandeur`pourrait être transformé en `user_is_not_demandeur` et ainsi de suite